wiki.py

```diff
@@ -553,3 +554,42 @@ def _add_year_categories(self):
                 else:
                     categories.append(extra_category)
             self._add_category_page(title, categories)
+
+    def _update_current_projects_template(self):
+        """Update the current projects template with the new projects.
+
+        All category pages are added to a category for the year. Some
+        are added to additional categories.
+
+        """
+
+        template_name = self._config["year_pages"]["current_projects_template"]
```

Comment:
Unless there is functionality missing, use the Template class instead.

Reply:
I think it was largely because I wasn't accustomed to using it. with `multiline_string()` the output should be almost the same so that should work.